docs/customization/plugin-api.md

```diff
@@ -147,7 +147,7 @@ const MySelectorPlugin = function(system) {
 }
 ```
 
-You can also use the Reselect library to memoize your selectors, which is recommended for any selectors that will see heavy use, since Reselect automatically memoizes selector calls for you:
+You can also use the Reselect library to memoize your selectors, which is recommended for any selectors that will see heavy use since Reselect automatically memoizes selector calls for you:
```

Comment:
```suggestion
You can also use the Reselect library to memoize your selectors, which is recommended for any selectors that will see heavy use, since Reselect automatically memoizes selector calls for you:
```